src/__test__/components/rating.components.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import StarRatings from 'react-star-ratings';
+import RateArticle, { mapStateToProps } from '../../components/Rating';
+
+const mockStore = configureStore([thunk]);
+const state = {
+  ratingReducer: {
+    averageRating: 2,
+    rating: 2,
+    ratingError: 'error',
+  },
+};
+const store = mockStore(state);
+
+describe('number of rating stars', () => {
+  it('renders two star rating components one for average rating and one for my rating', () => {
+    const wrapper = mount(<RateArticle store={store} />);
+    expect(wrapper.find(StarRatings).length).toEqual(2);
+  });
+  it('finds only my rating since the articles are undefined', () => {
```

Comment:
Let's have blank lines to separate each `it` block.
Apply this throughout your file

Reply:
Noted. I will rebase, refactor code and implement the changes

Reply:
Hound raises complains when a space is left after each it block